app/admin/lookup/controller.js

```diff
@@ -348,7 +348,7 @@ export default Ember.Controller.extend(BillingCategories, EKMixin,
       confirmDeleteValue(value) {
         let i18n = this.get('i18n');
         let title = i18n.t('admin.lookup.titles.deleteLookupValue');
-        let message = i18n.t('admin.lookup.messages.deleteLookupValue', { value });
+        let message = i18n.t('messages.delete_singular', { name: value.concat(' value') });
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.